modules/auth/auth.go

```diff
@@ -92,7 +91,7 @@ func (m *Module) parseToken(token string) (map[string]interface{}, error) {
 	tokenObj, err := jwt.Parse(token, func(token *jwt.Token) (interface{}, error) {
 		// Don't forget to validate the alg is what you expect:
 		if token.Method.Alg() != jwt.SigningMethodHS256.Alg() {
-			return nil, errors.New("Invalid signing method type")
+			return nil, errors.New("invalid signing method type")
```

Comment:
```suggestion
			return nil, errors.New("Invalid signing method type")
```

Reply:
But Go conventions say that `error` strings should not be capitalized @YourTechBud

Reply:
Makes sense. Cool. Let's discard this one